process_manager/views/partials.py

```diff
@@ -94,7 +97,35 @@ def process_table(request: HttpRequest) -> HttpResponse:
     )
 
 
+def handle_errors(
+    view_func: Callable[[HttpRequest], HttpResponse],
+) -> Callable[[HttpRequest], HttpResponse]:
+    """Decorator to handle errors.
+
+    Args:
+        view_func: The view function to be wrapped.
+
+    Returns:
+        The wrapped view function.
+    """
+    logger = logging.getLogger("django")
+
+    def wrapped_view(request, *args, **kwargs) -> HttpResponse:  # type: ignore
+        try:
+            return view_func(request, *args, **kwargs)
+        except Exception as e:
+            logger.exception(f"Error in view {view_func.__name__}: {e}")
```

Comment:
`logger.exception` will take care of recording all of the exception details so you don't need to include it in the error message.